### PR TITLE
Add docs for media_player.ps4

### PR DESCRIPTION
**Pull request by ktnrg45**
**Description:**
Added documentation for PS4 component. Added PS4 logo to images.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#19469

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/media_player.ps4.markdown [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+## {% linkable_title Configuration %}
+
+Add the following to your configuration.yaml file to add the PS4 component.
+
+```yaml
+# Example configuration.yaml entry
+media_player
+  - platform ps4
+    host 192.168.0.11
+    name Playstation 4
```
Comment (fabaff) [inline source/_components/media_player.ps4.markdown]: This needs to be valid YAML. 

We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps users to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

File: source/_components/media_player.ps4.markdown [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+```
+Configuration variables:
+
+- **host** (*Required*): The IP of the PS4 eg. `192.168.0.11`.
+- **name** (*Optional*): The name you would like to give to the PS4.
```
Comment (fabaff) [inline source/_components/media_player.ps4.markdown]: Please use the new style for the description of the configuration variables -> https://developers.home-assistant.io/docs/en/documentation_standards.html#component-and-platform-pages

**Review by fabaff — CHANGES_REQUESTED** [review]

File: source/_components/media_player.ps4.markdown (hunk 2 above)
Comment (ktnrg45) [reply source/_components/media_player.ps4.markdown]: Thanks for the info. Changes made.

File: source/_components/media_player.ps4.markdown [hunk 3]
```diff
@@ -0,0 +1,60 @@
+---
+layout: page
+title: "Playstation 4"
```
Comment (frenck) [inline source/_components/media_player.ps4.markdown]: Incorrect spelling of product title.

```suggestion
title: "PlayStation 4"
```

File: source/_components/media_player.ps4.markdown [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+description: "Instructions on how to integrate a Sony Playstation 4 into Home Assistant."
```
Comment (frenck) [inline source/_components/media_player.ps4.markdown]:
```suggestion
description: "Instructions on how to integrate a Sony PlayStation 4 into Home Assistant."
```

File: source/_components/media_player.ps4.markdown [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+- Android or iOS device
+- [PS4 Second Screen App](https://play.google.com/store/apps/details?id=com.playstation.mobile2ndscreen&hl=en_US) 
+
+After adding PS4 component to configuration.yaml, a configuration entity will be added to the frontend. This will fetch the credentials needed to allow Home Assistant to control the PS4.
+You will need to use the Second Screen app to do this.
+After credentials are fetched, a second configuration entity will be shown which allows you to pair Home Assistant to your PS4. 
+
+## {% linkable_title Granting Port Access %}
+
+The PS4 component requires the use of priveleged ports to work correctly. Depending on your OS of your Home Assistant instance you may need to allow usage of priveleged ports.
```
Comment (frenck) [inline source/_components/media_player.ps4.markdown]:
```suggestion
The PS4 component requires the use of privileged ports to work correctly. Depending on your OS of your Home Assistant instance you may need to allow usage of privileged ports.
```

File: source/_components/media_player.ps4.markdown [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+```yaml
+# Example configuration.yaml entry
+media_player:
+  platform: ps4
+    host: 192.168.0.11
+```
+{% configuration %}
+host: 
+  description: IP of the PS4 eg. `192.168.0.11`.
```
Comment (frenck) [inline source/_components/media_player.ps4.markdown]:
```suggestion
  description: IP of the PS4, e.g., `192.168.0.11`.
```

File: source/_components/media_player.ps4.markdown [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+  required: true
+  type: string
+name:
+  description: Name for PS4 entity.
+  required: false
+  type: string
+  default: Playstation 4
```
Comment (frenck) [inline source/_components/media_player.ps4.markdown]:
```suggestion
  default: PlayStation 4
```

File: source/_components/media_player.ps4.markdown [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+sharing: true
+footer: true
+logo: ps4.png
+ha_category: Media Player
+ha_release: 0.86
+ha_iot_class: "Local Polling"
+---
+
+The `ps4` component allows you to control a
+[Playstation 4](httpswww.playstation.com/en-us/explore/ps4/).
```
Comment (frenck) [inline source/_components/media_player.ps4.markdown]:
```suggestion
[PlayStation 4](httpswww.playstation.com/en-us/explore/ps4/).
```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @ktnrg45, reviewed it and left some comments. Could you please take a look? 👍

File: source/_components/media_player.ps4.markdown (hunk 8 above)
Comment (ktnrg45) [reply source/_components/media_player.ps4.markdown]: Ok. All typos should be corrected now.

**Comment by ktnrg45** [comment]
@fabaff @frenck  Changes made. Ready for review.

**Comment by ktnrg45** [comment]
Closed PR. Reopening.

**Pull request closed without merging**